hugin/gdocs_updater.py

```diff
@@ -99,7 +100,7 @@ def reshape_run_info(self, runs, skiplist=[]):
                 p = [p]
             for project in p:
                 if len(project) == 0:
-                    project = 'Unknown, please check!'
+                    continue
```

Comment:
@guillermo-carrasco: The run doesn't belong to any project (or project would be empty in description) when it couldn't find samplesheet for that run or for test runs where project field is not filled in the existing samplesheet.

Also in a rare case what Remi said :smile:  And @remiolsen  I don't think cards will be moved to "check_status" if the description is empty. As far as I understood, the script moves the card to "check_status" when the due-date for that card have passed. Again maybe I am wrong :)